lib/features/mailbox_dashboard/presentation/mailbox_dashboard_controller.dart

```diff
@@ -2,23 +2,21 @@ import 'package:flutter/material.dart';
 import 'package:flutter/widgets.dart';
 import 'package:get/get.dart';
 import 'package:jmap_dart_client/jmap/account_id.dart';
+import 'package:jmap_dart_client/jmap/core/id.dart';
 import 'package:jmap_dart_client/jmap/core/session/session.dart';
 import 'package:model/model.dart';
 import 'package:tmail_ui_user/features/base/base_controller.dart';
-import 'package:tmail_ui_user/features/mailbox/presentation/mailbox_controller.dart';
-import 'package:tmail_ui_user/features/thread/presentation/thread_controller.dart';
 
 class MailboxDashBoardController extends BaseController {
 
-  final mailboxController = Get.find<MailboxController>();
-  final threadController = Get.find<ThreadController>();
+  static final defaultAccountId = AccountId(Id('empty'));
 
   final scaffoldKey = GlobalKey<ScaffoldState>();
```

Comment:
`_scaffoldKey`

Reply:
In order for the view to be able to access it we need it not to be a private